api/server.py

```diff
@@ -324,40 +331,50 @@ def get_events_data(event_type):
 
     module_name = get_value_from_request("module_name")
     date = get_value_from_request("date")
-    filter = get_value_from_request("filter")
+    filter_by = get_value_from_request("filter")
 
     try:
-        query = filter_by_date(date) if date else {}
-        query.update(filter_by_module_name(module_name) if module_name else {})
-        query.update(
-            {
-                key: filter_by_regex(fix_filter_query(filter)[key]) for key in fix_filter_query(filter)
-            } if filter else {}
-        )
-
-        return jsonify(
-            {
-                "total": event_types[event_type].count(query),
-                "data": [
-                    i for i in
-                    event_types[event_type].find(
-                        query,
-                        {
-                            "_id": 0
-                        }
-                    ).skip(
-                        fix_skip(
-                            get_value_from_request("skip")
-                        )
-                    ).limit(
-                        fix_limit(
-                            get_value_from_request("limit")
-                        )
-                    )
-                ]
+        query = {
+            "query": {
```

Comment:
when the variable name is query then why do we need to mention inside the dict too?

Reply:
```
GET /_search
{
  "query" : {
     "match" : {
         "field": "value"   
    } 
  }
}
```
This is the basic structure of query in elasticsearch so that's why its needed.